### DISCUSSION TITLE
Need a pair of simple python scripts for http3 client and server

### DISCUSSION THREAD

**Discussion by pktCoder** [aiortc/aioquic#327] [Q&A]
Great repo.  So glad I find this when I was googling for HTTP3 implementation.  As part of a project, I need to create a simple HTTP3 GET transaction between two hosts, one running a client, one running a server. After that, both the client and server will exit. 

I am able to run the two scripts in the example directory: http3_client.py http3_server.py.   However, they are too complex. Wonder if someone could create a simpler version of them. 

For example,  restructure the http3_server.py so it doesn't use not use "Starlette",  just sends response in the class `HttpServerProtocol` method `http_event_received`. I tried the following but it didn't work.  Any help is appreciated.

```
self._http.send_headers(event.stream_id, [], False)
self._http.send_data(event.stream_id, b"hello world", True)
return
```

**Comment by devkdas** [comment]
Hai @pktCoder 

Hope this one helps

https://github.com/Karthikdasari0423/Basic-Aioquic

Thank you.

**Comment by pktCoder** [comment]
Thanks Karthikdasari0423!
